api/v1alpha1/halayerset_types.go

```diff
@@ -36,6 +36,9 @@ type HALayerSetSpec struct {
 
 	//NodesSpec contain names and ips of both SNO clusters nodes.
 	NodesSpec NodesSpec `json:"nodesSpec"`
+
+	//ContainerImage this image will be used by the HALayer pod to set up the pacemaker container which also contains the fencing agents - if left empty a default one will be used.
+	ContainerImage string `json:"containerImage,omitempty"`
```

Comment:
Not sure I agree.
I think it can be a bit misleading - basically the container is a Fedora which has an installed pacemaker as well as other things.
It is true that the pacemaker has a crucial role, but it is certainly not the only thing that is in the container.
[Here](https://github.com/mshitrit/pcmk/blob/703874f8d68d3bdf8c0755d61da55a9351e73ccc/Dockerfile) is a link to the docker file.

Reply:
> pacemaker has a crucial role

isn't this argument enough for a name containing pacemaker?
what do you when you need a second container later on?

Reply:
>isn't this argument enough for a name containing pacemaker?

I'm not against the name containing `pacemaker` however in this specific wording it implies that this is the only purpose of the container, which is wrong - for instance fencing is probably equally important and would likely be the main reason a user would like to configure this image in the first place.

>what do you when you need a second container later on?

I think it mainly depends on the purpose of the second container, for instance we can have something like LoadBalancingContainerInageURL for the first container (which includes `pacemaker` and `fence agents`) and MetricsContainerImageURL (in case this is what the second container is used for).
However I think this is completely theoretical, since there is a good that there will never be a second container so IMO I'd wait to cross that bridge when and if I'll get to it.

Reply:
I don't insist on exactly my suggestion, everything is better than just `ContainerImage`...

> this is completely theoretical

the CRD is our API, IMHO sometimes it might be good to also think about mostly theoretical future development...

Reply:
How about 
WorkloadManagerContainerImageURL ?